lib/context_adapter_helper.js

```diff
@@ -454,27 +535,41 @@ function getServiceDescriptor(request, operationDescriptor, callback) {
       {
         id: operationDescriptor.serviceId,
         type: caConfig.SERVICE_ENTITY.TYPE,
-        isPattern: false,
-        attributes: [
-          caConfig.SERVICE_ENTITY.ENDPOINT_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.METHOD_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.AUTHENTICATION_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.MAPPING_ATTR_NAME,
-          caConfig.SERVICE_ENTITY.TIMEOUT_ATTR_NAME
-        ]
+        isPattern: false
       }
+    ],
+    attributes: [
+      caConfig.SERVICE_ENTITY.INTERACTION_TYPE_ATTR_NAME,
+      caConfig.SERVICE_ENTITY.ENDPOINT_ATTR_NAME,
```

Comment:
Not really using it on the CA side but I do agree to include it since there are other attributes which are expected but not currently used... ;) Thanks, I'll include it in a sec.